e2e-tests/src/test/resources/appium-dev.example.properties

```diff
@@ -3,4 +3,6 @@
 #device.name=emulator-5554
 #appium.server.url=http://127.0.0.1:4723/wd/hub
 #demo.app.apk.path=../app/build/outputs/apk/debug/parasoft-demo-app-mobile-1.0.apk
-#pda.server.base-url=http://10.0.2.2:8080
\ No newline at end of file
+#pda.server.base-url=http://10.0.2.2:8080
+#app.package=com.parasoft.demoapp
```

Comment:
No need to add app.package as a config here, as it is a fixed value for the Android demo app.